doc/source/user_guide/changelog/v1.2.0.rst

```diff
@@ -0,0 +1,42 @@
+What's new in 1.2.0 (14 August 2024)
+-----------------------------------
+
+These are the changes in icepyx 1.2.0 See :ref:`release` for a full changelog
+including other versions of icepyx.
+
+
+New Features
+~~~~~~~~~~~~
+
+- Replace `setup.py` with equivalent `pyproject.toml` (#539)
+- Fix continuous delivery & docs to account for setup.py -> pyproject.toml change (#541)
+
+
+Bug fixes
+~~~~~~~~~
+
+- Switch to set comparison to fix a test dependent on unreliable order (#550)
+
+
+
+Maintenance
+^^^^^^^^^^^
+
+- update docstring tests for numpy 2.0 (#537)
+- Add Zenodo badge and update all-contributors badge (#536)
+- [pre-commit.ci] pre-commit autoupdate (#538)
```

Comment:
```suggestion
```

Reply:
Any particular reason?

Reply:
I prefer to only include things that can impact users in changelogs to optimize the "signal-to-noise" ratio. Us nerds who care about pre-commit can find that detail in the Git historY!

But if you prefer it in, I don't feel that strongly :)